docs/src/main/sphinx/sql/set-session.rst

```diff
@@ -15,28 +15,41 @@ Description
 
 Set a session property value or a catalog session property.
 
+.. _session-properties-definition:
 
-Connectors can provide session properties. They are set separately for each
-catalog by prefixing them with the catalog name. There is no sharing of
-properties across catalogs, even if the catalogs use the same connector.
+Session properties
+------------------
+
+A session property is a :doc:`configuration property </admin/properties>` that
+can be temporarily modified by a user for the duration of their current
+connection to the Trino cluster. Many configuration properties have a
```

Comment:
I *think* I understand "for the duration of their current connection," but do I? Is it until the current client or JDBC connection disconnects? Until the current user logs out but the connection remains open? Is it until the current cluster restarts? We *could* say more here, but I'm honestly not sure it's needed. It will be needed by newbies.

Reply:
It's important to be clear that it's not just for a given user, but for a given connection by that user. I *think* the current text does enough to clarify this alongside the last sentence of the section that describes disconnection behavior.

Reply:
Actually, I would like input on that last sentence, because it may be misleading to suggest that the configuration "resets" when you can have two parallel connections with different session configurations.